### ISSUE TITLE
Database update process failed: Forbidden; Blocked by CDN

### ISSUE THREAD

**Issue by kmooncare** [Cisco-Talos/clamav#862]
Describe the bug
----------------
Our production servers have Clamav installed in Alpine containers. We came to know that Clamav database has blocked our network due to outdated Clamav version. We discovered that the Clamav database had banned our network as a result of an out-of-date Clamav version.

How to reproduce the problem
----------------------------
When the Clamav service is up and operating inside the container, it will have blocked error problems and stop downloading updates from the database.

Attachments
-----------

`   | LibClamAV Warning: **************************************************
   | LibClamAV Warning: ***  The virus database is older than 7 days!  ***
   | LibClamAV Warning: ***   Please update it as soon as possible.    ***
   | LibClamAV Warning: **************************************************
   | Wed Mar 15 09:16:22 2023 -> ClamAV update process started at Wed Mar 15 09:16:22 2023
   | Wed Mar 15 09:16:22 2023 -> ^Your ClamAV installation is OUTDATED!
   | Wed Mar 15 09:16:22 2023 -> ^Local version: 0.103.2 Recommended version: 0.103.8
   | Wed Mar 15 09:16:22 2023 -> DON'T PANIC! Read https://www.clamav.net/documents/upgrading-clamav
   | Wed Mar 15 09:16:22 2023 -> daily database available for update (local version: 26365, remote version: 26842)
   | Wed Mar 15 09:16:23 2023 -> ^downloadPatch: Can't download daily-26366.cdiff from https://database.clamav.net/daily-26366.cdiff
   | Wed Mar 15 09:16:23 2023 -> ^Incremental update failed, trying to download daily.cvd
   | Wed Mar 15 09:16:23 2023 -> ^Can't download daily.cvd from https://database.clamav.net/daily.cvd
   | Wed Mar 15 09:16:23 2023 -> ^FreshClam received error code 403 from the ClamAV Content Delivery Network (CDN).
   | Wed Mar 15 09:16:23 2023 -> This could mean several things:
   | Wed Mar 15 09:16:23 2023 ->  1. You are running an out-of-date version of ClamAV / FreshClam.
   | Wed Mar 15 09:16:23 2023 ->     Ensure you are the most updated version by visiting https://www.clamav.net/downloads
   | Wed Mar 15 09:16:23 2023 ->  2. Your network is explicitly denied by the FreshClam CDN.
   | Wed Mar 15 09:16:23 2023 ->     In order to rectify this please check that you are:
   | Wed Mar 15 09:16:23 2023 ->    a. Running an up-to-date version of FreshClam
   | Wed Mar 15 09:16:23 2023 ->    b. Running FreshClam no more than once an hour
   | Wed Mar 15 09:16:23 2023 ->    c. If you have checked (a) and (b), please open a ticket at
   | Wed Mar 15 09:16:23 2023 ->       https://bugzilla.clamav.net under the 'Mirrors' component
   | Wed Mar 15 09:16:23 2023 ->       and we will investigate why your network is blocked.
   | Wed Mar 15 09:16:23 2023 -> !Database update process failed: Forbidden; Blocked by CDN
   | Wed Mar 15 09:16:23 2023 -> !Update failed.
   | Wed Mar 15 09:16:23 2023 -> ^FreshClam was forbidden from downloading a database.
   | Wed Mar 15 09:16:23 2023 -> ^This is fatal. Retrying later won't help. Exiting now.`

Additional Info
-----------------

We have upgraded the version by importing a upgraded container to the Server. As the server is blocked from your end, its not able to download the updates. Please lift the block so that we can solve the issue. 

Priority
-------
**Urgent.**

Since its a production server, we are almost blocked with respect to this issue. Please lift block so that we can solve the issue soon as possible.

**Comment by kmooncare** [comment]
@micahsnyder Please address soon as possible as its a higher priority for us.

**Comment by kmooncare** [comment]
@micahsnyder thanks for the info. Can you check the block status?

**Comment by micahsnyder** [comment]
After the rate limit expired we think we spotted a new issue and think we have fixed it. 

Please try again in a few minutes to see if it is now working.

**Comment by ramblak-commits** [comment]
https://bugzilla.clamav.net under the 'Mirrors' component
      and we will investigate why your network is blocked.

Thanks, Looking forward to being able to update.

**Comment by val-ms** [comment]
The 

> https://bugzilla.clamav.net under the 'Mirrors' component and we will investigate why your network is blocked.

0.103.2 is so old that it doesn't know we moved the issue tracker to github issues. 
This is the right place, however the reason you're being blocked is because 0.103 is past end of life for signature updates and you must upgrade. 

We currently support three feature versions: 1.0 LTS, 1.4 LTS, and 1.5 (non-LTS)

1.0.9 is the minimum that you would want. 1.4.3 is the best option for long term support. The 1.5.1 version just came out today.

**Comment by ramblak-commits** [comment]
Thank you. 


 

    On Thursday, October 16, 2025 at 12:18:49 p.m. EDT, Val S. ***@***.***> wrote:  
 
 val-ms left a comment (Cisco-Talos/clamav#862)
The


https://bugzilla.clamav.net under the 'Mirrors' component and we will investigate why your network is blocked.


0.103.2 is so old that it doesn't know we moved the issue tracker to github issues.
This is the right place, however the reason you're being blocked is because 0.103 is past end of life for signature updates and you must upgrade.

We currently support three feature versions: 1.0 LTS, 1.4 LTS, and 1.5 (non-LTS)

1.0.9 is the minimum that you would want. 1.4.3 is the best option for long term support. The 1.5.1 version just came out today.
